parsec/backend/invite.py

```diff
@@ -201,8 +201,11 @@ def generate_invite_email(
             use_ssl=backend_addr.use_ssl,
         )
     )
+    https_invite_link = parsec_invite_link.replace("parsec://", "")
+    split_invite_link = https_invite_link.split("/")
+    https_invite_link = "https://" + split_invite_link[0] + "/api/redirect/" + split_invite_link[1]
     body = [
-        line.format(sender_name=sender_name, invite_link=invite_link, parsec_url=parsec_url)
+        line.format(sender_name=sender_name, invite_link=parsec_invite_link, parsec_url=parsec_url)
```

Comment:
Here the `invite_link` need to be `https_invite_link` , not `parsec_invite_link` (this corresponds to the clickable button)